modules/tinymce/src/plugins/table/test/ts/browser/command/ApplyCellStyleCommandTest.ts

```diff
@@ -208,5 +208,70 @@ describe('browser.tinymce.plugins.table.command.ApplyCellStyleCommandTest', () =
     ]);
   });
 
+  context('border normalization', () => {
+    context('border normalization single cell', () => {
+      const testSingleCellNormalization = (initialStyles: string, styles: Record<string, string>, expectedStyles: string) => {
```

Comment:
A pattern that I've seen in a lot of other tests is something like this:
```typescript
Arr.each([
  {
    label: 'TINY-7593: should switch border-style to double when setting a border-colour on a 1px border-width cell',
    initialStyles: 'border: 1px solid red',
    styles: { borderColor: 'blue' },
    expectedStyles: 'border: 1px double blue'
  }, {
    // ..
  }, {
    // ...
  }
], (spec) => {
  it(spec.label, () => {
    const editor = hook.editor();
    const html = ...
  });
});
```

It's personal preference, but if you wanted to avoid the one-line tests it's an option.

Reply:
Good to know. It would make the test way more verbose though. But that seems like a good pattern to have. I think I stick with just looping the test guts for now.